linkis-orchestrator/linkis-computation-orchestrator/src/main/scala/org/apache/linkis/orchestrator/computation/conf/ComputationOrchestratorConf.scala

```diff
@@ -40,10 +40,7 @@ object ComputationOrchestratorConf {
     "rm,sh,find,kill,python,for,source,hdfs,hadoop,spark-sql,spark-submit,pyspark,spark-shell,hive,yarn"
   )
 
-  val SHELL_WHITE_USAGE = CommonVars(
-    "wds.linkis.shell.white.usage",
-    "sqoop,cd,ll,ls,echo,cat,tree,diff,who,grep,whoami,set,pwd,cut,file,head,less,if,while"
-  )
+  val SHELL_WHITE_USAGE = CommonVars("wds.linkis.shell.white.usage", "cd,ls")
 
```

Comment:
need update docs/configuration/linkis-computation-orchestrator.md  of wds.linkis.shell.white.usage  info